Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1540,6 +1540,13 @@
     Telecrystal: 8 # it's a gamble that may kill you easily so 4 TC per 2 uses, second one more of a backup
   categories:
     - UplinkImplants
+  #Sunrise-start
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)